### PR TITLE
Upgraded to webpack v2

### LINKED ISSUE #352: Upgrade webpack to v2
https://webpack.js.org/guides/migrating/

Because webpack v1 is deprecated.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #352

Changes made based on this migration guide: https://webpack.js.org/guides/migrating/

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -7,19 +9,25 @@ module.exports = {
...
+            options: {
+              presets: [`es2015`, `react`]
+            }
+          }
+        ]
       }
     ]
-  }
+  },
+  plugins: [
```
Comment (Pomax) [inline webpack.config.js]: Please uplift first, then add in more functionality to the config as follow-up. This is good to have but **only** for a production build. Source maps are actually super bloaty so it's far better to know which environment we're in and then either minify (for prod) without source maps, or not minify at all (for dev).

File: webpack.server.config.js [hunk 2]
```diff
@@ -23,19 +24,25 @@ module.exports = {
...
+            options: {
+              presets: [`es2015`, `react`]
+            }
+          }
+        ],
       }
     ]
-  }
+  },
+  plugins: [
```
Comment (Pomax) [inline webpack.server.config.js]: why would the server bundle need minification?

File: webpack.config.js (hunk 1 above)
Comment (mmmavis) [inline webpack.config.js]: kk will do

**Comment by mmmavis** [comment]
PR updated! I removed the `plugins` bits.

**Comment by Pomax** [comment]
looks good to me. Also filed https://github.com/mozilla/network-pulse/issues/359

**Comment by mmmavis** [comment]
Merged.

Thanks for filing that followup ticket! @Pomax

**Pull request merged**